prow/config/config.go

```diff
@@ -216,14 +216,36 @@ type InRepoConfig struct {
 	AllowedClusters map[string][]string `json:"allowed_clusters,omitempty"`
 }
 
+func trimRepoPrefix(repo string) string {
+	repo = strings.TrimPrefix(repo, "http://")
+	repo = strings.TrimPrefix(repo, "https://")
+	return repo
+}
+
+func SplitRepoName(repo string) (string, string, error) {
+	// Normalize repo name to remove http:// or https://, this is the case for some
+	// of the gerrit instances.
+	repo = trimRepoPrefix(repo)
+	s := strings.SplitN(repo, "/", 2)
+	if len(s) != 2 {
+		return "", "", fmt.Errorf("repo %s cannot be split into org/repo", repo)
+	}
+	return s[0], s[1], nil
+}
+
 // InRepoConfigEnabled returns whether InRepoConfig is enabled for a given repository.
+// Assumes that config will not include http:// or https://
 func (c *Config) InRepoConfigEnabled(identifier string) bool {
-	if c.InRepoConfig.Enabled[identifier] != nil {
-		return *c.InRepoConfig.Enabled[identifier]
+	normalizedIdentifier := trimRepoPrefix(identifier)
+
+	if c.InRepoConfig.Enabled[normalizedIdentifier] != nil {
+		return *c.InRepoConfig.Enabled[normalizedIdentifier]
 	}
-	identifierSlashSplit := strings.Split(identifier, "/")
-	if len(identifierSlashSplit) == 2 && c.InRepoConfig.Enabled[identifierSlashSplit[0]] != nil {
-		return *c.InRepoConfig.Enabled[identifierSlashSplit[0]]
+
+	// Errors if failed to split. We are ignoring this and just checking if org != "" instead
+	org, _, _ := SplitRepoName(identifier)
+	if org != "" && c.InRepoConfig.Enabled[org] != nil {
```

Comment:
org is not used outside of this if/else condition, should combine
```suggestion
	if org, _, _ := SplitRepoName(identifier); org != "" && c.InRepoConfig.Enabled[org] != nil {
```

Reply:
Good catch. Fixed